docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,15 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
+
+- If you rely on wrapping arithmetic, surround each operation with ``unchecked { ... }``.
+- Optional: If you use SafeMath or a similar library, change ``x.add(y)`` to ``x + y``, ``x.mul(y)`` to ``x * y`` etc.
+- Add ``pragma abicoder v1;`` if you want to stay with the old ABI coder.
```

Comment:
```suggestion
- Add ``pragma abicoder v1;`` if you want to stay with the old ABI coder. Optionally remove ``pragma experimental ABIEncoderV2`` or ``pragma abicoder v2`` since it is redundant.
```

Reply:
I'd put it the extra part in a separate bullet point.

Reply:
Fixed the suggestion!